src/common/hdfs/HdfsCommandHelper.cpp

```diff
@@ -49,6 +50,7 @@ Status HdfsCommandHelper::copyToLocal(const std::string& hdfsHost,
     folly::Subprocess proc(std::vector<std::string>({"/bin/bash", "-c", command}));
     auto result = proc.wait();
     if (!result.exited()) {
+      LOG(INFO) << "Failed to download: " << result.str();
```

Comment:
Here download error will failed this job too?

Reply:
Yeah, if download occurs error we should fail it. (Exchange will not generate empty dir, if it indeed generate empty dir, ingest will handle it)